testing/test_assertrewrite.py

```diff
@@ -70,10 +70,16 @@ class TestAssertionRewrite(object):
     def test_place_initial_imports(self):
         s = """'Doc string'\nother = stuff"""
         m = rewrite(s)
-        # Module docstrings in 3.7 are part of Module node, it's not in the body
+        # Module docstrings in 3.8 are part of Module node, it's not in the body
```

Comment:
do we know for sure it will change in 3.8, or do we need a tracking issue?

Reply:
Not sure, let's ask @ncoghlan

Reply:
We're not sure yet what's going to happen in 3.8, as the key point that we didn't take into account this time around is the significant difference that moving module docstrings around creates between doing `ast.parse("'Hello'", mode="exec").body[0].value` and `ast.parse("'Hello'", mode="eval").body`.

I *suspect* what we'll end up going with is Naoki Inada's idea of introducing a dedicated `DocString` node into the AST, since that still allows a bunch of special case heuristics in the compiler to be tidied up (since they can dispatch directly on node type instead), without losing the location information for the docstring, and without making them disappear from the node tree entirely.